Docs/source/usage/examples.rst

```diff
@@ -61,6 +61,17 @@ Uniform plasma
 :download:`2D case <../../../Examples/Physics_applications/uniform_plasma/inputs_2d>`
 :download:`3D case <../../../Examples/Physics_applications/uniform_plasma/inputs_3d>`
 
+Capacitive discharge
+--------------------
+
+The Monte-Carlo collision (MCC) model can be used to simulate capacitive discharges between parallel plates. The implementation has been tested against the benchmark results from Turner et. al. in `Phys. Plasmas 20, 013507, 2013 <https://aip.scitation.org/doi/abs/10.1063/1.4775084>`_. The figure below shows a comparison of the ion density as calculated in WarpX (in June 2021) compared to the literature results (which can be found `here <https://aip.scitation.org/doi/suppl/10.1063/1.4775084>`__). An input file with parameters for the 1st case is given below except the total simulation steps have been reduced.
+
+.. figure:: MCC_benchmark_Turner2013.png
```

Comment:
Can you try if this works?
```suggestion
.. figure:: https://user-images.githubusercontent.com/40245517/121434798-b0ff4800-c932-11eb-9a34-ef6225b543d7.png
```

Alternatively, we can try to use `.. image::`

Reply:
Great idea! I tried it and it does work. I've had the third test case from literature running for a while now and it should finish tomorrow afternoon. When it does I'll add those results and update the plot in the PR description. I'll then also remove the plot from the PR and update the link in the docs.

Reply:
When you do, you'll need to rebase and edit the commits that have the plots to exclude them, so that they don't become part of the merged history

Reply:
@PhilMiller I believe they said they always squash-merge PRs, so it should get eliminated at that point anyway.